image/scanner/rhel/konflux.Dockerfile

```diff
@@ -18,8 +20,8 @@ RUN unzip -j blob-repo2cpe.zip -d image/scanner/dump/repo2cpe && \
     unzip -j blob-k8s-definitions.zip -d image/scanner/dump/k8s_definitions && \
     unzip -j blob-nvd-definitions.zip -d image/scanner/dump/nvd_definitions
 
-RUN echo -n "version: " && scripts/konflux/version.sh && \
-    go build -trimpath -ldflags="-X github.com/stackrox/scanner/pkg/version.Version=$(scripts/konflux/version.sh)" -o image/scanner/bin/scanner ./cmd/clair
+RUN echo -n "version: " && make --quiet --no-print-directory tag && \
+    make CGO_ENABLED=1 scanner-build-nodeps
```

Comment:
There will be now `2.33.x-80-gbef2d6709f-fast` in `github.com/stackrox/scanner/pkg/version.Version` and that one is used in multiple places. I'm not sure what's going to break but at least let's break it early enough and see.